alerts.go

```diff
@@ -48,7 +50,7 @@ var commandAlerts = cli.Command{
 				},
 				cli.BoolTFlag{Name: "color, c", Usage: "Colorize output. default: true"},
 				cli.BoolFlag{Name: "with-closed, w", Usage: "Display open alert including close alert. default: false"},
-				cli.IntFlag{Name: "limit, l", Value: 100, Usage: "Set the number of alerts to display at once when withClosed is active. default: 100"},
+				cli.IntFlag{Name: "limit, l", Value: defaultAlertsLimit, Usage: "Set the number of alerts to display."},
```

Comment:
`mkr alerts -help` describes default value of limit as 100, but actually it's not 100 but all when `--with-closed` is false.
Can you adjust help description?
```
   --limit value, -l value  Set the number of alerts to display. (default: 100)
```